src/sentry/static/sentry/app/components/projectSelector.jsx

```diff
@@ -79,11 +85,16 @@ class ProjectSelector extends React.Component {
   }
 
   getProjects() {
-    const {organization, projects} = this.props;
-    const projectList =
-      projects || organization.projects.filter(project => project.isMember);
+    const {organization, projects, multiProjects} = this.props;
+    const {isSuperuser} = ConfigStore.get('user');
```

Comment:
this might be overkill. I don't know why anyone would want superuser powers on the old dropdown.